hedwiki/score/HED_score_0.0.2.mediawiki

```diff
@@ -20,10 +20,10 @@ This project was supported by the by the National Institute of Mental Health of
         ** <nowiki># {takesValue,valueClass=numericClass, unitClass=frequencyUnits}</nowiki>
     * Hyperventilation
         ** Quality-of-hyperventilation
-            *** Refused-procedure
-            *** Poor-effort
-            *** Good-effort
-            *** Excellent-effort
+            *** Hyperventilation-refused-procedure
```

Comment:
Would these be better phrased as:
Refused-hyperventilation-procedure
Poor-hyperventilation-effort     .... etc.